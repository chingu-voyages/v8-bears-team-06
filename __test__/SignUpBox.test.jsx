import { shallow } from "enzyme";
import React from "react";
import renderer from "react-test-renderer";

import SignUp from "../client/components/SignUpBox";

describe("With Enzyme", () => {
  it('SignUpBox shows "Sign Up"', () => {
    const signup = shallow(<SignUp />);

    expect(signup.find("h1").text()).toEqual("Sign Up");
  });
});

describe("With Snapshot Testing", () => {
  it('SignUpBox shows "Sign Up"', () => {
    const component = renderer.create(<SignUp />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
