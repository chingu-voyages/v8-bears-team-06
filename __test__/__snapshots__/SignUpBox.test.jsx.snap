// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With Snapshot Testing SignUpBox shows "Sign Up" 1`] = `
<form
  className="jsx-678892718 form-signin mt-5"
>
  <h1
    className="jsx-678892718 mb-3"
  >
    Sign Up
  </h1>
  <div
    className="jsx-678892718 form-group"
  >
    <label
      className="jsx-678892718 float-left required"
      htmlFor="email"
    >
      Email address
    </label>
    <input
      autoFocus={true}
      className="jsx-678892718 form-control"
      id="email"
      onChange={[Function]}
      placeholder="Enter your email"
      required={true}
      type="email"
      value=""
    />
    <small
      className="jsx-678892718 form-text text-muted float-left mb-3"
      id="emailHelp"
    >
      We'll never share your email with anyone else.
    </small>
    <br
      className="jsx-678892718"
    />
  </div>
  <div
    className="jsx-678892718 form-group"
  >
    <label
      className="jsx-678892718 float-left required"
      htmlFor="password"
    >
      Password
    </label>
    <input
      className="jsx-678892718 form-control"
      id="password"
      onChange={[Function]}
      placeholder="Enter your password"
      required={true}
      type="password"
      value=""
    />
    <small
      className="jsx-678892718 form-text text-muted float-left mb-3"
      id="emailHelp"
    >
      Must contain at least 6 alphanumeric characters.
    </small>
    <br
      className="jsx-678892718"
    />
  </div>
  <button
    className="jsx-678892718 btn btn-primary float-left"
    onClick={[Function]}
    type="submit"
  >
    Sign in
  </button>
</form>
`;
